ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -276,9 +277,9 @@ export const ApplicationSummary = (props: {app: models.Application; updateApp: (
                     {urls
                         .map(item => item.split('|'))
                         .map((parts, i) => (
-                            <a key={i} href={parts.length > 1 ? parts[1] : parts[0]} target='__blank'>
+                            <Link key={i} to={parts.length > 1 ? parts[1] : parts[0]} target='__blank'>
```

Comment:
Sorry, I've just realized that links a line 280 and 361 contains absolute URLs (typically external URL) and don't need to be updated. Can you please revert them back to `<a>`

Reply:
thanks for noticing, reverted those to a tags